sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobServiceVersion.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob;
+
+import com.azure.core.http.ServiceVersion;
+
+/**
+ * The versions of Azure App Configuration supported by this client library.
+ */
+public enum BlobServiceVersion implements ServiceVersion {
+    V2019_02_02("2019-02-02");
```

Comment:
What string does the service expect to see? Is it prefixed with `V`, and the blob code is doing that? What string do the version docs state - with or without the `V` prefix. It just feels like this is a little disconnected.

Reply:
The expected header format is just "YYYY-MM-DD"; no 'v' or anything else.

Reply:
In this case I wonder if we should name the enum values with a prefix of `V` (as we have here), or if we should just use the string as-is. I'm not convinced the `V` really adds any value.

Reply:
As discussed offline, keeping the V prefix to stay consistent with .NET.